lib/protocol/rest_json.js

```diff
@@ -4,6 +4,16 @@ var Json = require('./json');
 var JsonBuilder = require('../json/builder');
 var JsonParser = require('../json/parser');
 
+function unsetContentLength(req) {
+  var payloadMember = util.getRequestPayloadShape(req);
+  if (
+    payloadMember === undefined &&
+    ['GET', 'HEAD', 'DELETE'].indexOf(req.httpRequest.method) >= 0
+  ) {
+    delete req.httpRequest.headers['Content-Length'];
```

Comment:
is this ever any other case than `Aaaa-Aaaa`?

Reply:
Not that I know of. What's `Aaaa-Aaaa` case?